Add Mate-Limo (2.Versuch)

tad3u5 opened the pull request:
Fixes: #25
Also hier das Rezept zur Selbstbau **Hackerbrause.**

tad3u5 committed:
Grundstruktur erstellt

tad3u5 committed:
eine Mate bitte 

DIE Hackerbrause darf nicht fehlen

SanJSp reviewed (changes requested):
Hallo @tad3u5 

vielen Dank für deinen Beitrag. Ich würde dich trotzdem bitten dich an die Formattierungsrichtlinien aus dem Beispielrezept zu halten. Bitte füge z.B. Überschriften ein.

tad3u5 committed:
Überarbeitung des Darstellung

das Rezept wurde dem Repostil angepasst und mit kleinen Änderungen im Text versehen.

verspaeteterZug commented on Getränke/Limonade/Mate-Limo.md:
```diff
@@ -0,0 +1,21 @@
+
+## Zutaten
+
++ ***6 TL loser Mate-Tee (1 TL gespricht ca. 2g)***
```
Ich möchte mit diesem Kommentar eigentlich nur checken,wie die
+ ***Formatierung***
+ ***aussieht***

tad3u5 replied:
Das + gibt die Aufzählung und die drei * führen zu dicker und kursiver Schreibweise.
